static/js/containers/PostPage.js

```diff
@@ -140,6 +155,48 @@ class PostPage extends React.Component<*, void> {
     )
   }
 
+  showRemoveCommentDialog = (comment: CommentInTree) => {
+    const { dispatch } = this.props
+    dispatch(setModeratingComment(comment))
+    dispatch(showDialog(DIALOG_REMOVE_COMMENT))
+  }
+
+  hideRemoveCommentDialog = () => {
+    const { dispatch } = this.props
+    dispatch(clearModeratingComment())
+    dispatch(hideDialog(DIALOG_REMOVE_COMMENT))
+  }
+
+  removeComment = async () => {
+    const { dispatch, removingComment } = this.props
```

Comment:
`removingComment` is sort of like the focused comment, right? I might suggest rename to something like `commentToRemove` or something along those lines, the `removingComment` name to me sounds a bit more like something telling us about the status (currently removing vs no).